LICENSE.txt

```diff
@@ -16,6 +16,49 @@ reference.
 
 -------
 
+Some math approximations are provided by:
+/*=====================================================================*
+ *                   Copyright (C) 2011 Paul Mineiro                   *
+ * All rights reserved.                                                *
+ *                                                                     *
+ * Redistribution and use in source and binary forms, with             *
+ * or without modification, are permitted provided that the            *
+ * following conditions are met:                                       *
+ *                                                                     *
+ *     * Redistributions of source code must retain the                *
+ *     above copyright notice, this list of conditions and             *
+ *     the following disclaimer.                                       *
+ *                                                                     *
+ *     * Redistributions in binary form must reproduce the             *
+ *     above copyright notice, this list of conditions and             *
+ *     the following disclaimer in the documentation and/or            *
+ *     other materials provided with the distribution.                 *

```

Comment:
This license doesn't belong in LICENSE.txt. If you search for "Redistributions" in the git, you'll see hundreds of other files are covered by similar licenses and they do get a mention in the LICENSE.txt:

"The dRonin code is licensed under the GPLv3. There are a few minor exceptions
to this so please see the headers of all source code for copyright and license
information."

Nothing more necessary. Code is open source, and binaries link to the public source code, which qualifies as "other materials provided".

Just keep the 3-clause in the misc_math.h file